installing.md

```diff
@@ -292,13 +283,12 @@ reindex all your data after making schema changes.
 Now that the schema is set up, you're ready to begin importing data!
 
 Our [goal](https://github.com/pelias/pelias/issues/255) is that eventually you'll be able to run all
-the importers with simply `cd $importer_directory; npm start`. Unfortunately only the Who's on First
-and OpenStreetMap importers works that way right now.
+the importers with simply `cd $importer_directory; npm start`. We are now really close, and all but
+one importer follows this pattern!
```

Comment:
I'm being finicky, but don't like describing it changes as "special", as it can be vague